crates/env/src/backend.rs

```diff
@@ -290,6 +290,13 @@ pub trait EnvBackend {
         output: &mut [u8; 20],
     ) -> Result<()>;
 
+    fn sr25519_verify(
```

Comment:
Please add a comment in the style of the other functions above.

Reply:
done